### DISCUSSION TITLE
Per world homes remover

### DISCUSSION THREAD

**Discussion by ehetteNandaYo** [EssentialsX/Essentials#4088] [Ideas and Feature Suggestions] [answered]
Basically a command that will run through all the players files and delete their homes in X world
Could also work per player

Reason: World resets.

**Comment by pop4959** [comment]
There is a plugin linked on Discord that will do this for you (pinned in #ess-support): https://cdn.discordapp.com/attachments/426460575971672064/758165938972393523/EssentialsXRemoveAllHomesWithoutDeletingAllPlayerdataExceptIfItsJason-1.0.1.jar

**Reply by triagonal** [reply]
Just a heads up that the linked plugin will delete /all/ of your player homes. You can't choose which world it'll operate on if you only wanted them deleted in a specific world.

**Reply by ehetteNandaYo** [reply]
Thanks but I need per world, I want to keep my player main world homes.

**Comment by triagonal** [comment, marked as answer]
This has now been implemented as an `/essentials` sub-command. Run `/essentials homes` with no arguments to view all home cleanup options. To delete homes in a specific world, use `/essentials homes delete <worldname>`. **This action is irreversible!** 



Download the latest dev build at https://essentialsx.net/downloads.html.
